src/main/webapp/user-page.html

```diff
@@ -47,7 +53,7 @@ <h1 id="page-title">User Page</h1>
 
     <div id="message-container">Loading...</div>
     <form id="message-form" action="/messages" method="POST" class="hidden">
-      Enter a new message:
+      <div id="update-text">Enter a new message:</div>
```

Comment:
You aren't supposed to have multiple elements with the same id in a page. Typically you'd use a class instead of an id for something like this. In this particular case though this looks like a second level header so you should probably use an h2 tag here as well.